test/distributed/_composable/fsdp/test_fully_shard_training.py

```diff
@@ -243,7 +243,6 @@ def world_size(self) -> int:
         return min(8, torch.cuda.device_count())
 
     @skip_if_lt_x_gpu(2)
-    @test_compiled_fsdp()
```

Comment:
remove trivial use ucase and instead focusing on `torch.compile(Transformer)` w/o AC